app-benchmarks/geekbench/geekbench-2.4.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A Cross-Platform Benchmark for Android, iOS, Linux, MacOS and Windows"
+HOMEPAGE="https://www.geekbench.com"
+SRC_URI="https://cdn.primatelabs.com/Geekbench-${PV}-Linux.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="geekbench"
+SLOT="2"
+
+RDEPEND="sys-devel/gcc
```

Comment:
Why no RDEP in 2.4.2?

Reply:
ldd says, 2.4.2 for arm is static. 2.4.3, 3.4.2 and 4.2.2 are not static.

Reply:
Well, that's good enough argument for me. From my experience static executables rarely work cross-libc (contrary to what people using them think) but not our problem here.

Reply:
Actually, `readelf -d` lists the libraries for me (in *_arm_32).

Reply:
I see, checked myself, I've added RDEPEND.